app/src/app-logic.js

```diff
@@ -1,16 +1,22 @@
-import React, { useCallback, useMemo, useState } from 'react'
-import { useAragonApi, useConnectedAccount } from '@aragon/api-react'
+import React, { useCallback } from 'react'
+import {
+  AragonApi,
+  useApi,
+  useAppState,
+  useCurrentApp,
+} from '@aragon/api-react'
+import appStateReducer from './app-state-reducer'
 import usePanelState from './hooks/usePanelState'
-import useUnwrapPanelState from './hooks/usePanelState'
```

Comment:
It's generally bad practice re-import a module under a different name. For hooks, we can just re-use the hook twice with a different variable assignment.